databricks/koalas/base.py

```diff
@@ -32,6 +32,8 @@
 from databricks.koalas.internal import _InternalFrame
 from databricks.koalas.typedef import pandas_wraps, spark_type_to_pandas_dtype
 from databricks.koalas.utils import align_diff_series, scol_for
+from databricks.koalas.internal import SPARK_INDEX_NAME_FORMAT
```

Comment:
This import seems not used.

Reply:
i just removed. thanks for commenting! :D